scrapy/commands/genspider.py

```diff
@@ -61,6 +62,12 @@ def run(self, args, opts):
             raise UsageError()
 
         name, domain = args[0:2]
+
+        parsed_url = urlparse(domain)
+
+        if not parsed_url.scheme:
+            parsed_url._replace(scheme='http')
```

Comment:
```suggestion
            parsed_url = parsed_url._replace(scheme='http')
```
This is because `_replace` does not modify the object in-place, it returns a new `ParseResult` instead.